src/EdgeAnalysis.jl

```diff
@@ -22,4 +22,49 @@ end
 
 export TerminationTime
 export TerminationConversion
-export TerminationRateRatio
\ No newline at end of file
+export TerminationRateRatio
+
+"""
+Generate appropriate core Simulation/SystemSimulation object
+"""
+function tosim(sol,domains,inters,p)
+    return Simulation(sol,domains,inters,p)
+end
+
+"""
+Generate appropriate core Simulation/SystemSimulation object
+"""
+function tosim(sol,domains::Tuple,inters,p)
+   return SystemSimulation(sol,domains,inters,p) 
+end
+
+"""
+Generate appropriate edge Simulation/SystemSimulation object
+"""
+function getsim(inte,react,coreedgedomain,domain,inters,p,coretoedgespcmap)
+    ycoreedge = getycoreedge(inte.u,coretoedgespcmap,coreedgedomain.indexes[end])
+    sol = build_solution(react.ode,inte.alg,[0.0,inte.t],[ycoreedge,ycoreedge])
+    sim = Simulation(sol,coreedgedomain,inters,p)
+    return sim
+end
+
+"""
+Generate appropriate edge Simulation/SystemSimulation object
+"""
+function getsim(inte,react,coreedgedomains::Tuple,domains::Tuple,inters,p,coretoedgespcmap)
```

Comment:
Input `domains` isn't used in this function?

Reply:
Thanks!